azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/lobby/ConnectingLobbyOverlayView.kt

```diff
@@ -51,6 +59,14 @@ internal class ConnectingLobbyOverlayView : LinearLayout {
             }
         }
 
+        viewLifecycleOwner.lifecycleScope.launch {
+            viewModel.getCallingStatusStateFlow().collect {
+                if (it == CallingStatus.RINGING) {
```

Comment:
can use flow.filter instead of if, for a more fluent api usage.

Reply:
As per discussion we will fix for GA.